src/vs/platform/browserView/electron-main/browserView.ts

```diff
@@ -242,6 +249,18 @@ export class BrowserView extends Disposable {
 						metaKey: input.meta || false,
 						repeat: input.isAutoRepeat || false
 					});
+				} else if (isEscapeWhileLoading) {
```

Comment:
Let's just add Esc to the keycodes we bubble up. I think we'll want other shortcuts using it too, like cancelling element selection